src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/reissuefurtherevidence/ReissueFurtherEvidenceAboutToSubmitHandlerTest.java

```diff
@@ -159,18 +163,21 @@ public void doesNotReturnAnErrorIfNoSelectedOriginalSender() {
 
     @Test
     public void returnsAnErrorIfThereIsNoPartySelectedToReIssueFurtherEvidence() {
-        sscsCaseData = sscsCaseData.toBuilder().resendToRepresentative("NO").resendToDwp("NO").resendToAppellant("NO").build();
+        ReissueArtifactUi reissueFurtherEvidence = sscsCaseData.getReissueArtifactUi();
+        reissueFurtherEvidence.setResendToAppellant(YesNo.NO);
+        reissueFurtherEvidence.setResendToRepresentative(YesNo.NO);
+        reissueFurtherEvidence.setResendToDwp(YesNo.NO);
 
         when(caseDetails.getCaseData()).thenReturn(sscsCaseData);
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
         assertEquals(1, response.getErrors().size());
-        assertEquals("Select a party to reissue the further evidence.", response.getErrors().toArray()[0]);
+        assertEquals("Select a party to reissue.", response.getErrors().toArray()[0]);
```

Comment:
Doesn't the original error message make more sense because there is context?

Reply:
I spoke to Femi and she agreed with the change to make it more generic so we can reuse the same code on different places